syntax = "proto2";

package bus;

message BusRequest {
  required int32 id = 1;
  required int32 number = 2;
}

message BusResponse {
    required int32 id = 1;
    required int32 number = 2;
  
    required int32 people_capacity = 3;
    required float average_people_count = 4;
}

service Dyonysus{
   rpc Service(BusRequest) returns (BusResponse) {}
}